src/ebonite/build/builder/base.py

```diff
@@ -32,6 +32,20 @@ def use_local_installation():
         EBONITE_FROM_PIP = tmp
 
 
+@contextmanager
+def use_wheel_installation(path: str):
+    """Context manager that changes docker builder behaviour to
+    install ebonite from wheel.
+    This is needed in the case you using ebonite from wheel"""
+    global EBONITE_FROM_PIP
```

Comment:
maybe rename it to smth like _EBONITE_SOURCE and write some docs on which value yield which result